topic-3.3/fileManagerTest.java

```diff
@@ -0,0 +1,44 @@
+package bootcamp33.topic3_3;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
+import bootcamp33.topic3_3.FileManager;
+
+public class fileManagerTest {
+
+	@Test
+	public void test() {
+		fail("Not yet implemented");
+		int limit = 2;
+		List<File> files = new ArrayList<File>();
+		FileManager fileM = new FileManager(limit);
+		files = fileM.getList();
+		assertEquals(files.size(), 0);
+
+		fileM.open("C:\\Users\\Ciel\\aux1.txt");
+		fileM.close();
+		files = fileM.getList();
+		assertEquals(files.size(), 1);
+
+		fileM.open("C:\\Users\\Ciel\\aux1.txt");
+		fileM.close();
+		files = fileM.getList();
+		assertEquals(files.size(), 1);
```

Comment:
[High] You are not testing when the file already exists in the list it is bumped to the top. The class method is fine, but you need to demonstrate you have applied TDD.
Perhaps you can add another file aux2 and then add aux1 again and finally get first element to ensure it is aux1.